test/signs/realtime_test.exs

```diff
@@ -1406,9 +1424,20 @@ defmodule Signs.RealtimeTest do
       Signs.Realtime.handle_info(:run_loop, sign)
     end
 
+    test "Station with routes is closed" do
+      sign = %{
+        @mezzanine_sign_with_routes
+        | tick_read: 0
+      }
+
+      expect_messages({"No Orange Line", "Service ended for night"})
+      expect_audios(canned: {"103", ["883"], :audio})
```

Comment:
`883` reads out "This station is closed. Service has ended for the night.", but the ticket says it should now read "Orange Line service has ended for the night". Luckily, we already have take `882` that reads just "service has ended for the night" which can combine with takes that we have for the route names. We'll have to start passing the routes to `%Content.Audio.ServiceEnded{}` though and call a helper within its `to_params/1` implementation for the bidirectional case.

Reply:
Oh nice catch! How does this look? https://github.com/mbta/realtime_signs/pull/759/commits/a799010fd5209f50bdd875b8594fec321d9f6851

Reply:
Overall seems right! But I think `routes` will always be a list though so could we get away with removing the call to `List.wrap/1`? 

I'm also a little torn on whether or not we should default to using `@service_ended`. The `Utilities.line_to_var()` helper defaults to using the take for "Train" which I think could be fine.

Reply:
@PaulJKim okay, updated to remove `List.wrap` and defaulting to `@service_ended`: [`a799010..137cc1d`](https://github.com/mbta/realtime_signs/compare/a799010fd5209f50bdd875b8594fec321d9f6851..137cc1d427477a8d484281bc1d150d259030d33a?w=1)